src/wp-admin/admin-header.php

```diff
@@ -33,7 +33,10 @@
 }
 
 get_admin_page_title();
-$title = strip_tags( $title );
+
+if ( ! is_null( $title ) ) {
+	$title = wp_strip_all_tags( $title ); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+}
```

Comment:
Since `wp_strip_all_tags()` now handles `null` (see [related trac ticket](https://core.trac.wordpress.org/ticket/56434) and commit d46dc08342c8e90e16289ae850097d6a31118cd6), I suppose we could do without the extra check, and only replace `strip_tags` by `wp_strip_all_tags`:

```suggestion
$title = wp_strip_all_tags( $title ); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
```

Reply:
HIya @jeherve !

IMO, the pulled fix is not the right approach/fix for the following reason: `$title` may still be `null` afterwards.
This means, the problem is now moved to subsequent functions which may be handling `$title` and using PHP native functions to do so, so this is just setting WP up for the next bug report instead of fixing the actual issue that `$title` should always be set and set to a `string`.

As for the second proposal (comment above), which I do think is better, there are two things I'd like to point out about that:
1. Yes, it will handle `null` silently, but for anything scalar but non-string, that will now yield a `warning`, which is a significantly higher error level than the original `deprecation` (though justifiable).
2. The `// phpcs:ignore ...` comment needs to be removed as that WPCS rule does not apply to WP Core and we should not add unnecessary noise like that to the codebase.